tests/test.c

```diff
@@ -75,6 +75,35 @@ struct Test
 #define SKIP_STACKTRACES false
 #endif
 
+// Enabling this will override malloc and calloc weak symbols,
+// so we can force an alternative version of malloc that returns
+// NULL when size is 0.
+// This is useful to find debugging or finding some kind of issues.
+#ifdef FORCE_MALLOC_ZERO_RETURNS_NULL
+// required for memalign
```

Comment:
AFAIK, memalign is a deprecated linuxism. Standard function is posix_memalign.

Reply:
Good point, fixed.